app/views/katello/api/v2/docker_manifest_lists/show.json.rabl

```diff
@@ -4,7 +4,7 @@ attributes :id, :schema_version, :digest, :manifest_type
 
 child :docker_tags => :tags do
   attributes :associated_meta_tag_identifier => :id
-  attributes :repository_id, :name
+  attributes :repository, :name
```

Comment:
is it possible to do this without changing the api?

something like  attributes :repository => :repository_id

Reply:
@jlsherrill yeah this works fine.